File: Aula6/ex1.cpp
#include <iostream>
using namespace std;

int reverso(int numero) {
    int reverso = 0;
    while (numero != 0) {
        int numeross = numero % 10; 
        reverso = reverso * 10 + numeross; 
        numero /= 10; 
    }
    return reverso;
}

int main() {
    int numero;
    cout << "Digite um número inteiro: ";
    cin >> numero;

    int resultado = reverso(numero);
    cout << "O reverso do número é: " << resultado << endl;

    return 0;
}

File: Aula4/ex1/ex1.cs
/*Exercício 1) C#
Fazer um programa para ler um número
inteiro, e depois dizer se este número é par
ou não.*/


using System;
class Program
{
    static void Main()
    {
        Console.Write("Digite um número inteiro: ");
        int numero = int.Parse(Console.ReadLine());

        if (numero % 2 == 0)
        {
            Console.WriteLine("O número é par.");
        }
        else
        {
            Console.WriteLine("O número não é par.");
        }
    }
}